server/src/main/java/org/apache/iotdb/db/mpp/execution/operator/process/SingleInputAggregationOperator.java

```diff
@@ -91,9 +83,8 @@ public ListenableFuture<?> isBlocked() {
     return child.isBlocked();
   }
 
-  @Override
-  public boolean hasNext() {
-    return curTimeRange != null || timeRangeIterator.hasNextTimeRange();
+  protected boolean hasMoreData() {
```

Comment:
This method seems to be only used in RawDataAggregationOperator.

Reply:
I've moved it to `RawDataAggregationOperator `.